ui/components/app/assets/token-cell/token-cell.test.tsx

```diff
@@ -90,7 +92,17 @@ describe('Token Cell', () => {
     image: '',
     onClick: jest.fn(),
   };
-  (useSelector as jest.Mock).mockReturnValue(MOCK_GET_TOKEN_LIST);
+  const useSelectorMock = useSelector;
+  (useSelectorMock as jest.Mock).mockImplementation((selector) => {
+    if (selector === getTokenList) {
+      return MOCK_GET_TOKEN_LIST;
+    }
+    if (selector === getMultichainCurrentChainId) {
+      return '0x89';
+    }
+    return undefined;
+  });
+  // useSelector.mockReturnValue(MOCK_GET_TOKEN_LIST);
```

Comment:
is this comment relevant ? we should delete it

Reply:
woops, thanks for catching. will remove

Reply:
https://github.com/MetaMask/metamask-extension/pull/26994/commits/e6b567e22f7f3cb3227a752dde7f109db9b17266